File: 8 Kyu Python/Is there a vowel in there_.py
def is_vow(inp):
    vowels = {97: 'a', 101: 'e', 105: 'i', 111: 'o', 117: 'u'}
    result = []
    for num in inp:
        if num in vowels:
            result.append(vowels[num])
        else:
            result.append(num)  
    return result

File: 8 Kyu JavaScript/Is there a vowel in there_.js
function isVow(a){
  const vowels = {
    97: 'a',
    101: 'e',
    105: 'i',
    111: 'o',
    117: 'u'
  };
  const result = [];
  for (let i = 0; i < a.length; i++) {
    const num = a[i];
    if (vowels.hasOwnProperty(num)) {
      result.push(vowels[num]);
    } else {
      result.push(num);
    }
  }
  return result;
}